src/tables/sortable-table.js

```diff
@@ -147,7 +148,7 @@ function SortableTable({
   }
 
   return (
-    <table className={classnames(className, { 'sortable-table': !disableSort })}>
+    <table className={classnames(className, { 'sortable-table': !disableSort })} {...rest}>
```

Comment:
Can you use the `filterInvalidDOMProps` util here (and a corresponding test)?

Reply:
@chawes13 : thanks for taking the time to review this!!